gwsumm/data/coherence.py

```diff
@@ -432,14 +434,38 @@ def get_coherence_spectrograms(channel_pairs, segments, config=None,
     return out
 
 
+def _get_common_segments(segments, spans):
```

Comment:
I'm still not convinced why this function is need at all, since functionally it just does

```python
reduce(operator.and_, segments, spans).coalesce()
```

but with some coalescing up-front.

Reply:
@duncanmmacleod, I've updated this to handle the call to `reduce` in-block, without the extra coalescing up-front.